docs/src/terminology.md

```diff
@@ -100,7 +100,7 @@ When nodes representing 2/3rd of the stake have a common [root](terminology.md#r
 
 ## fork
 
-A [ledger](terminology.md#ledger) derived from common entries but then diverged.
+A portion of a [ledger](terminology.md#ledger) that has diverged from a common entry when compared to another validator’s view of the ledger.
```

Comment:
The original definition seems more precise.  How about just adding a sentence such as, "Forking occurs when validators vote on different blocks at the same slot height."